docs/index.md

```diff
@@ -0,0 +1,11 @@
+## Introduction
+
+OpenBLAS is an optimized Basic Linear Algebra Subprograms (BLAS) library based on [GotoBLAS2](https://www.tacc.utexas.edu/research-development/tacc-software/gotoblas2) 1.13 BSD version.
+
+OpenBLAS implements low-level routines for performing linear algebra operations such as vector addition, scalar multiplication, dot products, linear combinations, and matrix multiplicationon. OpenBLAS makes these routines available on multiple popular platforms, covering desktop (Windows, Linux, Mac)  and mobile (Android, iOS) operating systems, as well as different architectures (namely x86 and ARM).
```

Comment:
"matrix multiplication." (typo). And I'd like to leave out the "popular" here, plus make it "server, desktop and mobile" and change the "namely x86 and ARM" to something like "including ARM, MIPS, PPC, RISC-V, x86 and zarch"